docker-compose.yml

```diff
@@ -105,7 +110,7 @@ services:
       args:
         - GIT_SHA
         - LOCALE_ENV
-    image: mozmeao/kitsune:full-${GIT_COMMIT_SHORT:-latest}
+    image: mozmeao/kitsune:full-${CI_COMMIT_SHORT_SHA:-latest}
```

Comment:
I think we should not create the variable name specific to CI. better to have the previous general name like `GIT_COMMIT_SHORT`

Reply:
except these are specifically for CI. Shouldn't need anything other than "latest" locally.

Reply:
This can be used locally also! sometime you may need to run specific version of the image, in that case you can set this environment variable and run that version locally!